prefect_databricks/flows.py

```diff
@@ -326,7 +327,7 @@ async def jobs_runs_submit_by_id_and_wait_for_completion(
     pipeline_params: Optional[str] = None,
     sql_params: Optional[Dict] = None,
     dbt_commands: Optional[List] = None,
-    **jobs_runs_submit_kwargs: Dict[str, Any],
+    **jobs_runs_submit_kwargs,
```

Comment:
Is there a reason why the typing was changed?

Reply:
Reverted back. That was unintended